src/backend/marsha/core/tests/test_views_lti_cache.py

```diff
@@ -126,13 +127,11 @@ def test_views_lti_cache_instructor(self, mock_get_consumer_site, mock_verify):
         }
 
         with self.assertNumQueries(3):
-            elapsed, resource_origin = self._post_lti_request(url, data)
+            _, resource_origin = self._post_lti_request(url, data)
```

Comment:
Note that you can name a variable `_elapsed` so you keep the semantic and make it optional in pylint's eyes.